### ISSUE TITLE
Train and score oracle micro-relational transitions

### ISSUE THREAD

**Issue by Sino-Huang** [Sino-Huang/NovPhy#5]
## Parent

#1 — BG-NS-JEPA model and experimental design

## What to build

Train the micro mode from oracle micro predicates and carry it end to end through carrier prediction, supervised readout, exhaustive scoring, and immutable pair artifacts.

## Acceptance criteria

- Micro-relational content can change the predicted carrier, not only an auxiliary loss.
- Micro readouts are supervised against oracle semantics.
- Eligible micro pairs appear in exhaustive score and frontier artifacts.
- Continuous rollout state remains the sole carrier.

## Blocked by

#2, #3, #4

**Comment by Sino-Huang** [comment]
The cohort-v2 micro handoff is now available.

Authority is `cohort-v2-micro-relation-derivation-spec-v1:contact+supports` through the v5 release reader. Both predicates are fixed-step, entity-ID relation sets; `contact` is symmetric and `supports` is directed. Each carries explicit availability, and unavailable records must be masked rather than trained/scored as empty relations or negatives. Older proposal references to velocity-bin micro predicates are not supplied by the approved central cohort.

Training parameters may be learned only from the `training` exposure role. Calibration and model-selection roles retain their narrower permissions; ordinary readers cannot access final evaluation.

**Comment by Sino-Huang** [comment]
Completed and pushed issue #5.

Commits:
- implementation: 57c1f120f5c7a0eece4abce34bac92e02cf43622
- compact evidence: 50e2104

Acceptance evidence:
- micro contact/support content enters the selected transition adapter and changes the continuous carrier; directed support and symmetric contact contracts are covered;
- the public micro logits and exact entity-pair relation queries are both supervised from cohort-v2 oracle semantics, with unavailable predicates masked;
- exhaustive evaluation emitted 5,354 states and all 48,186 grid outcomes: 32,124 available continuous/micro outcomes and 16,062 explicitly unavailable macro outcomes;
- all three micro horizons are available in the model-selection frontier artifact, 1,600 states each;
- rollout chaining remains continuous-carrier only;
- checkpoint v2 binds learned tensor bytes as sha256:da71e8d1c3f92787b16fd284981040d55a04abb1bdf90cef1d00102850709263;
- learned parameters used only the training exposure role; ordinary readers did not access final evaluation.

Verification:
- python -u -m scripts.run_cohort_v2_micro_experiment --validate — passed;
- focused model/cohort gate — 92 passed;
- final Standards review — clean;
- final Spec review — clean.

Committed evidence: data/runtime_evidence/issue-5/cohort-v2-micro-experiment-summary.json. The checkpoint and full 48,186-outcome artifacts remain ignored under .local-artifacts/issue-5-micro-experiment.
